easy-ui-react/package.json

```diff
@@ -31,7 +31,9 @@
   "dependencies": {
     "@easypost/easy-ui-icons": "1.0.0-alpha.2",
     "@easypost/easy-ui-tokens": "1.0.0-alpha.1",
+    "@react-aria/toast": "^3.0.0-beta.0",
```

Comment:
unless im missing something, it seems like the `toast` hooks aren't part of their monopackage yet for react-aria: https://github.com/adobe/react-spectrum/blob/main/packages/react-aria/src/index.ts my guess is that the `toast` hooks are still in beta